multiload/properties.c

```diff
@@ -652,7 +652,6 @@ fill_properties(GtkWidget *dialog, MultiloadApplet *ma)
 	gtk_container_set_border_width (GTK_CONTAINER (page), 12);
 	add_color_selector(page, _("_User"), "memload-color0", ma);
 	add_color_selector(page, _("Sh_ared"), "memload-color1", ma);
-	add_color_selector(page, _("_Buffers"), "memload-color2", ma);
```

Comment:
Why do you skip a number for a compound identifier?

Reply:
We want to keep the color names and purposes intact: "color3" for cache and "color4" for "free". We do not want to change it and have "color3" become the color of "free", otherwise users will be surprised their assigned color for "free" (black, usually) is now not black for some reason.

Reply:
That is the last question we should decide about: should we do this kinda messy thing with a dummy color and passing a zero to keep the colors from "sliding", or do we mess up the colors and release this as a major update that requires reassigning the colors?

Reply:
The user can see the color labeling in the preferences dialog.

Reply:
They can, but if they upgrade to this version from a previous version while having this applet already on their panel, then the colors will change, and the user will have to reassign them. Which is undesirable, arguably enough to have this workaround.

Reply:
If you delete a data source you already modify the graph.

Reply:
We delete a data source that was already almost invisible. And we can avoid messing up the other colors with this workaround. I am not insisting that it is a good idea - I leave that decision to others.

Reply:
Modify the color assignment.

Reply:
Which color assignment are you talking about?

If you mean "modify the part where the colors are assigned to the graph", then I can not even find that part, and I think that would not be worth the effort for others to do, either.

If you mean that you vote for "clean code, messy upgrade", then it's now 1:1. :)

Reply:
Take a look at multiload/org.mate.panel.applet.multiload.gschema.xml.in

Reply:
> They can, but if they upgrade to this version from a previous version while having this applet already on their panel, then the colors will change, and the user will have to reassign them. On an already configured applet that was not supposed to change! Which is undesirable, arguably enough to have this workaround.

As i said  before , this is a massive change and not a simple bug fix.
https://github.com/mate-desktop/mate-applets/pull/464#issuecomment-657233868
So this will not be in stable before next MATE major release which is 1.26, IHMO.
That means that will not be available in your actual distro release.
I think it is OK to change colors of users configs in next distro release.
Or does someone consider this as a simple bug fix which we can push in stable 1.24 of current distro releases?
I am i worry that users will blame us about missing buffer memory.

Reply:
I think this is fine to put in the next major release.

@lukefromdc was the only one in favor of trying to preserve a smooth upgrade. So, do we drop the last two commits?